Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py

```diff
@@ -105,12 +119,21 @@ def get_sample(client: Client, **args) -> CommandResults:
 
 
 def get_sample_summary(client: Client, **args) -> CommandResults:
-    sample_id = args.get("sample_id")
-    r = client._http_request("GET", f"samples/{sample_id}/summary")
+    outputs = []
+    for sample_id in argToList(args.get("sample_id", "")):
```

Comment:
Isn't there an option to make the API call one time for all IDs?

Reply:
There isn't such an API, no. Each ID is a separate analysis.